packages/utilities/src/guards/PermissionGuard/PermissionGuard.ts

```diff
@@ -94,6 +88,9 @@ export function PermissionGuard(
       return next();
     }
 
-    return post(arg, messageIfNotAllowed);
+    if (options.content instanceof MessageEmbed)
```

Comment:
2 things:

1. please use brackets, please run the linter,
2. `else` is not necessary

```ts
if (options.content instanceof MessageEmbed){
	return post(arg, { embeds: [options.content], ephemeral: options.ephemeral ? true : false });
}
  
return post(arg, { content: options.content, ephemeral: options.ephemeral ? true : false });
```
is enough

Reply:
Please can this actually be refactored? 

```ts
const postOptions = {ephemeral: options.ephemeral ? true : false };
if(options.content instanceof MessageEmbed){
   postOptions["embeds"] = [options.content];
}else{
    postOptions["embeds"] = options.content;
}
return post(arg, postOptions);
```